src/config.rs

```diff
@@ -32,6 +32,10 @@ impl Config {
 
         let mut conf = Config::from_str(&buffer).unwrap();
 
+        if let Some(_) = conf.rest.get_mut("book") {
+            panic!("Found a '[book]' section on your 'mdzk.toml' file. You might want to replace it with '[mdzk]' ;-)")
```

Comment:
Neither a panic, nor an error-return is necessary here, as the config will load just fine with a `[book]` section. You can use the `warn!` macro here to throw a warning message to the user